plugins/inputs/postgresql/postgresql.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"strings"
 
+	// Black import required to register driver
```

Comment:
Maybe `blank` would be more appropriate? :)

Reply:
:facepalm: thanks for catching this... Fixed.